templates/zerver/emails/followup_day1.source.html

```diff
@@ -5,51 +5,55 @@
 {% endblock %}
 
 {% block content %}
+{% if realm_creation %}
+{% if is_demo_org %}
+<p>
+    {% trans %}Congratulations, you have created a new Zulip organization. Note
+    that this organization will be automatically deleted in 30 days. Learn more
+    about demo organizations <a
+    href="https://zulip.com/help/demo-organizations">here</a>!
```

Comment:
This should be using the `{% trans apps_page_link="https://zulip.com/apps" %}` style of syntax for the link -- this helps keep the URLs and syntax out of what we ask translators to translate.

I think we might also want to call it a "new Zulip demo organization"?